sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client.py

```diff
@@ -16,6 +16,9 @@
 from phone_number_testcase import PhoneNumberCommunicationTestCase
 from _shared.testcase import BodyReplacerProcessor
 
+SKIP_PHONE_NUMBER_TESTS = True
```

Comment:
Just wondering if there is a way for us to have this defined somewhere at global level rather than doing it in every file?

Reply:
we could use an environment variable for this too, but we are going to get rid of this soon anyway.
Once the LRO is ready, we will do on-demand phone acquisition. Thus, there will be no need to skip